package.json

```diff
@@ -45,7 +45,7 @@
     "chai": "^4.3.6",
     "klona": "^2.0.5",
     "mocha": "^10.0.0",
-    "vc-test-suite-implementations": "github:w3c/vc-test-suite-implementations"
+    "vc-test-suite-implementations": "../vc-test-suite-implementations"
```

Comment:
you keep using a local version of vc-test-suite-implementations. have you tried `npm link`?

Reply:
Sorry that keeps sneaking in. I'll give `npm link` a look--and clean this out again.